glide.yaml

```diff
@@ -18,7 +18,9 @@ import:
 - package: github.com/uber-go/atomic
   version: ^1
 - package: github.com/prometheus/client_golang
-  version: ^1
+  version: 1.1
+- package: github.com/prometheus/procfs
+  version: 0.0.6
```

Comment:
Newer versions of Prom client have dependencies on libraries using go modules with v2, which does not work with glide. Since this is legacy anyway, I am pinning to the current versions (unfortunately glide doesn't allow updating a single dep).